src/components/BaseListGroupItemAssetPool.vue

```diff
@@ -1,10 +1,20 @@
 <template>
     <b-list-group-item
         :to="`/memberships/${membership.id}/withdrawals`"
-        v-if="membership && membership.token"
         class="d-flex justify-content-between align-items-center"
     >
-        <div class="mr-auto">
+        <div v-if="!membership.token" class="mr-auto">
+            <i
+                class="fas fa-code-branch mr-2"
+                :class="{ 'text-success': membership.network, 'text-muted': !membership.network }"
```

Comment:
@GarfDev I dont think membership.token will be set appropriately when the GET /memberships/:id request returns a 404. In other words, this will always display the gray icon, no matter the actual network the membership was in. 

Better to remove this and use a different icon to indicate an issue. Maybe this one?
```
<i class="fas fa-exclamation-circle"></i>
```